unison-core/src/Unison/HashQualified'.hs

```diff
@@ -19,7 +20,7 @@ import qualified Unison.ShortHash              as SH
 import qualified Unison.HashQualified          as HQ
 
 data HashQualified n = NameOnly n | HashQualified n ShortHash
-  deriving (Eq, Functor, Generic)
+  deriving (Eq, Functor, Generic, Foldable)
```

Comment:
Came in handy at one point - the `toList` will give you just the name, ignoring the short hash if that exists.